joint_limits_interface/package.xml

```diff
@@ -0,0 +1,39 @@
+<package format="2">
+  <name>joint_limits_interface</name>
+  <version>0.16.0</version>
+  <description>Interface for enforcing joint limits.</description>
+
+  <maintainer email="bence.magyar.robotics@gmail.com">Bence Magyar</maintainer>
+  <maintainer email="enrique.fernandez.perdomo@gmail.com">Enrique Fernandez</maintainer>
+  <maintainer email="mathias.luedtke@ipa.fraunhofer.de">Mathias Lüdtke</maintainer>
```

Comment:
Add not-active maintainers as authors

Reply:
can you suggest me which are the authors and maintainer of this package?

Reply:
I think Enrique and Mathias are not active so much here, so I would move them in authors. @bmagyar is clearly maintainer :)